tests/ui-toml/large_include_file/too_big.txt

```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ultricies mi quis hendrerit dolor. Sit amet tellus cras adipiscing enim eu turpis egestas. Faucibus nisl tincidunt eget nullam non nisi est. Ipsum faucibus vitae aliquet nec ullamcorper sit. Maecenas pharetra convallis posuere morbi leo urna. Purus in mollis nunc sed id semper. Sit amet tellus cras adipiscing enim eu turpis egestas. Mauris rhoncus aenean vel elit scelerisque mauris pellentesque. Massa vitae tortor condimentum lacinia quis vel.
```

Comment:
Unrelated, but I wanted to share my thoughts ^^. You can resolve this :upside_down_face: 

Lorem ipsum, the blind text of all blind texts. I would have probably spent 20+ minutes just searching for a text that can be included. There are so many possibilities to choose from. We could even write a poem about Clippy, the brave paperclip. That's just the inner troll in me *Lorem ipsum* is a good choice IMO.

Reply:
A Clippy poem would've been great. Wish I was creative enough for something like that :smile: